components/scream/src/physics/zm/zm_conv.F90

```diff
@@ -552,12 +540,11 @@ subroutine zm_convr(lchnk   ,ncol    , &
    integer ii
    integer k
    integer msg                      !  ic number of missing moisture levels at the top of model.
-   integer ierror
 
    real(r8) qdifr
    real(r8) sdifr
 
-   logical :: is_first_step
+   logical :: is_first_step = .false.
```

Comment:
@AaronDonahue , please look at this line in particular. This logical was being used uninitialized.